### PR TITLE
[dev16] consume Roslyn async packages

### PR DESCRIPTION
**Pull request by TIHan**
This is the work @brettfo has been doing and is almost ready for merging.

Will need to update the RoslynPackageVersion.txt.

### CODE REVIEW

File: vsintegration/src/FSharp.Editor/Options/EditorOptions.fs [hunk 1]
```diff
@@ -57,10 +57,10 @@ type AdvancedOptions =
     { IsBlockStructureEnabled: bool 
       IsOutliningEnabled: bool }
 
-[<Export(typeof<ISettings>)>]
```
Comment (majocha) [inline vsintegration/src/FSharp.Editor/Options/EditorOptions.fs]: The reason this was exported and initialized independently was to handle the following scenario:
The user opens Visual Studio and without opening any solution goes directly to Tools -> Options -> Text Editor -> F#
FSharpPackage.Initialize() was not called in such case. I'm not sure how would async package behave. It is something we need to ensure still works.

Also, I regret not commenting this properly.

**Comment by cartermp** [comment]
test Ubuntu16.04 Release_default Build

**Comment by cartermp** [comment]
Link #5363

**Comment by brettfo** [comment]
@dotnet-bot test this

**Comment by brettfo** [comment]
@dotnet-bot test Windows_NT Release_ci_part1 Build

**Comment by brettfo** [comment]
@dotnet-bot test Ubuntu16.04 Release_default Build

**Pull request merged**